src/inventory.cpp

```diff
@@ -560,11 +560,6 @@ u32 InventoryList::getUsedSlots() const
 	return num;
 }
 
-u32 InventoryList::getFreeSlots() const
-{
-	return getSize() - getUsedSlots();
-}
```

Comment:
Looks potentially useful.

Reply:
Whereas it might look like that, it's a very trivial function. For fitting items, `roomForItem` is used in various places and is more reliable than this check.